src/components/form/field-controls/radio-field/docs.mdx

```diff
@@ -0,0 +1,60 @@
+---
+componentName: RadioField
+peerComponents:
+	- ProductIcon
+---
+
```

Comment:
An introductory summary of the purpose of this component and how it is commonly used would be great!